package.json

```diff
@@ -15,40 +15,42 @@
   },
   "contributors": [],
   "dependencies": {
-    "asset-wrap": "0.9.x",
-    "async": "0.9.x",
-    "chokidar": "0.11.x",
+    "async": "2.6.0",
+    "chokidar": "2.0.1",
     "coffeelint": "1.x.x",
-    "coffee-script": "1.7.x",
-    "colors": "0.6.2",
-    "columnify": "1.2.1",
+    "coffeescript": "1.7.x",
+    "colors": "1.1.2",
+    "columnify": "1.5.4",
     "connect": "2.17.x",
-    "copy-paste": "0.3.x",
+    "copy-paste": "1.3.x",
+    "dobi-asset-wrap": "2.0.2",
+    "dobi-mongofb": "1.3.0",
     "express": "3.4.x",
     "findit": "1.2.x",
-    "firebase": "1.0.x",
+    "firebase": "2.2.9",
```

Comment:
you can bump this up to the very last 2.x release (i think `2.4.2`) - just adds some bug fixes and promises (but still supports callbacks)